docs/package.json

```diff
@@ -80,10 +81,11 @@
     "eslint-config-universe": "^10.0.0",
     "eslint-plugin-lodash": "^7.3.0",
     "http-server": "^0.12.3",
-    "jest": "26.1.0",
+    "jest": "^27.4.7",
```

Comment:
thanks for updating deps

Reply:
I kinda wanted it to test the code 😬  It works pretty well, especially with the newer `jest.mocked` method!